src/AElf.Playground.Application/Jobs/JobsAppService.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Volo.Abp.Application.Services;
+
+namespace AElf.Playground.Jobs
+{
+    public class JobsAppService(
+        JobManager jobManager) : ApplicationService
+    {
+        private readonly JobManager _jobManager = jobManager;
+
+        public async Task CreateJobAsync(Guid id, string command)
+        {
+            if (command == "build")
+            {
+                await _jobManager.BuildAsync(id);
+            }
+            else if (command == "test")
+            {
+                await _jobManager.TestAsync(id);
+            }
+            else
+            {
+                throw new Exception($"Unknown command {command}.");
```

Comment:
Can throw UserException instead of Exception since this is for the user i presume.